src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,311 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures aggregation expression performance for aggregations that
+  can take in multiple expressions as arguments.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE variadic aggregation expressions and numbers on the 'standalone'
+  environment for the classic variadic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 16
+  IntFields10: &IntFields10 ["$int1", "$int2", "$int3", "$int4", "$int5", "$int6", "$int7", "$int8", "$int9", "$int10"]
+  IntFields20: &IntFields20 ["$int1", "$int2", "$int3", "$int4", "$int5", "$int6", "$int7", "$int8", "$int9", "$int10", "$int11", "$int12", "$int13", "$int14", "$int15", "$int16", "$int17", "$int18", "$int19", "$int20"]
+
+Actors:
+# Clear any pre-existing collection state.
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: *Threads
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          int1: {^RandomInt: {min: -100, max: 100}}
```

Comment:
As per offline discussion, since we have only 10 fields, I chose not to use the ^cycle generator. Let me know if you think I should use it and reduce the number of fields we insert in a document.

Reply:
There's two questions here:
- How many args should we have? 5 and 10 seem better, but I'm not sure whether something like 3 and 6 or 3 and 5 would be more realistic. I've pinged Chris and Katya on slack to see if they have any opinions; if they don't let's stick with with 5 and 10
- What should the document schema look like? The current schema feels a bit crazy, but it's probably for the best as a "realistic" use case wouldn't duplicate field arguments. However, one (very optional) suggestion: could you try and use the [object generator to cut down on some of the redundancy here?](https://github.com/mongodb/genny/blob/dcd3a0527cce7afbe1aaec90b3cb488672253b5e/src/workloads/docs/Generators.yml#L236) If not, it's no big deal, but it could make this more parseable.